File: examples/v2/policy-issue/go/main.go
package main

import (
	v2 "github.com/pulumi/pulumi-pulumiservice/sdk/go/pulumiservice/v2"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		cfg := config.New(ctx, "")
		organizationName := cfg.Get("organizationName")
		if organizationName == "" {
			organizationName = "service-provider-test-org"
		}
		issueId := cfg.Get("issueId")
		if issueId == "" {
			issueId = "example-issue-id"
		}

		_, err := v2.NewPolicyIssue(ctx, "issue", &v2.PolicyIssueArgs{
			OrgName:    pulumi.String(organizationName),
			IssueId:    pulumi.String(issueId),
			Priority:   pulumi.String("high"),
			Status:     pulumi.String("in_progress"),
			AssignedTo: pulumi.String("pulumi-bot"),
		})
		return err
	})
}

File: examples/v2/policy-issue/python/__main__.py
import pulumi
import pulumi_pulumiservice.v2 as ps_v2

config = pulumi.Config()
organization_name = config.get("organizationName") or "service-provider-test-org"
issue_id = config.get("issueId") or "example-issue-id"

ps_v2.PolicyIssue(
    "issue",
    org_name=organization_name,
    issue_id=issue_id,
    priority="high",
    status="in_progress",
    assigned_to="pulumi-bot",
)
